node/heartbeat/errors.go

```diff
@@ -58,3 +58,6 @@ var ErrUnmarshalGenesisTime = errors.New("monitor: can't unmarshal genesis time"
 
 // ErrMarshalGenesisTime signals that the marshaling of the genesis time didn't work
 var ErrMarshalGenesisTime = errors.New("monitor: can't marshal genesis time")
+
+// ErrPropertyTooLong signals that one of the properties is too long
+var ErrPropertyTooLong = errors.New("Property Too Long in Heartbeat")
```

Comment:
error messages should begin with a lowercase letter

Reply:
renamed to "property too long in Heartbeat"